camayoc/tests/qpc/api/v1/credentials/test_network_creds.py

```diff
@@ -24,6 +24,7 @@
 from camayoc.utils import uuid4
 
 
+@pytest.mark.skip(reason="Expects hardcoded /sshkeys/ locally and at the server")
```

Comment:
I wonder the reason wouldn't be clear enough for me if I didn't knew what's going on. This small addition might help

```suggestion
@pytest.mark.skip(reason="Expects the same hardcoded /sshkeys/ locally and at the server")
```

Also, to avoid repeating the same text I'd put it in a variable (but that's just my personal taste :stuck_out_tongue: )

```suggestion
SKIP_REASON_LOCAL_SSHKEYS = "Expects the same hardcoded /sshkeys/ locally and at the server"

@pytest.mark.skip(reason=SKIP_REASON_LOCAL_SSHKEYS)
```

Reply:
I accepted your suggestions, I'm rebasing/amending the changes and pushing.